src/modules/Transcript/Header/styles.css

```diff
@@ -0,0 +1,3 @@
+.transcript__header-search {
+    @apply mr-2;
```

Comment:
Why not just have this inline?

Reply:
good call, it was initially part of a different css block, but removed now